scaffolding-chef-infra/lib/linux/scaffolding.sh

```diff
@@ -83,6 +83,17 @@ do_default_install() {
   bootstrap_chunk=$(cat "${lib_dir}"/bootstrap-chunk.rb)
   echo -e "${shared_chunk}\n${bootstrap_chunk}" >> "${pkg_prefix}/config/bootstrap-config.rb"
 
+  if $scaffold_report_on_install; then
+    if [ -f "$PLAN_CONTEXT/default.toml" ]; then
+      echo `grep "^chef_guid" "$PLAN_CONTEXT/default.toml" | sed 's/^chef_guid *= */chef_guid /'` >> "${pkg_prefix}/config/bootstrap-config.rb"
```

Comment:
I don't think this is the best way to implement this we shouldn't be grepping the default.toml for these values instead we should be using a variable like `scaffold_automate_server_url`. An example of this can be seen in the `scaffolding-chef-inspec`

Reply:
I didn't even realize that was a thing, I'll take a look at the `scaffolding-chef-inspec` example you provided and update my PR to behave like that